src/handlers/messages.ts

```diff
@@ -324,6 +380,58 @@ export async function handle(rcMessage: RcMessage): Promise<void> {
     return
   }
 
+  const ts = new Date(rcMessage.ts.$date).valueOf()
+  if (rcMessage.file) {
+    if (rcMessage.attachments?.length == 1) {
```

Comment:
Why just a length of one? I haven't seen anything with more than one attachment, but we could loop over it anyhow.

Reply:
Because the name of the file is taken from `rcMessage.file._id` which there is only one of.
I imagine if the `rcMessage.attachments` has a greater length, all attachments would use the same file name. So it should be better to explicitly warn for this case.

Reply:
Indeed, I do agree